src/routes.js

```diff
@@ -26,6 +27,7 @@ export const getRoutes = (store) => {
       <Route path="/signup" component={SignupContainer} />
       <Route path="/home" component={HomeContainer} onEnter={authRequired}/>
       <Route path="/login" component={LoginContainer} />
+      <Route path="/target" component={Target} />
```

Comment:
I think that `onEnter={authRequired}` is missing here. The path `/target` don't need authentication?